### PR TITLE
Locale requirement and fix motd when IP is not in PATH

### PR DESCRIPTION
**Pull request by ElieDeloumeau**

### CODE REVIEW

File: scripts/00-basic [hunk 1]
```diff
@@ -13,7 +13,7 @@ PROCESS_USER=$(echo "$PROCESS"| grep -v root | awk {'print $2'} | awk '{ SUM +=
 PROCESSOR_NAME=$(grep "model name" /proc/cpuinfo | cut -d ' ' -f3- | awk {'print $0'} | head -1)
 PROCESSOR_COUNT=$(grep -ioP 'processor\t:' /proc/cpuinfo | wc -l)
 
-LOCAL_IP_ADDRESSES=$(ip address | grep -E "inet(6)? " | grep -v 127.0.0.1 | grep -v 'inet6 ::1' | awk '{print $2;}' | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/\n\                /g')
+LOCAL_IP_ADDRESSES=$(/usr/sbin/ip address | grep -E "inet(6)? " | grep -v 127.0.0.1 | grep -v 'inet6 ::1' | awk '{print $2;}' | sed -e ':a' -e 'N' -e '$!ba' -e 's/\n/\n\                /g')
```
Comment (xp-1000) [inline scripts/00-basic]: the path will change depending on the distribution.

hardcoding the absolute path is a bad idea.

I am curious to know in which case `ip` is not available in `PATH` ?

**Review by xp-1000 — CHANGES_REQUESTED** [review]

File: scripts/00-basic (hunk 1 above)
Comment (ElieDeloumeau) [reply scripts/00-basic]: Do you know a distribution for which the command is elsewhere?

On Debian, when you are logged in as user, you don't have /usr/sbin in PATH (by default)

Comment (xp-1000) [reply scripts/00-basic]: archlinux

Comment (ElieDeloumeau) [reply scripts/00-basic]: I removed IPs list. This script is not intended to be more complicated ¯\_(ツ)_/¯

Comment (xp-1000) [reply scripts/00-basic]: what about making this field optional by checking if command is available?

**Review by xp-1000 — APPROVED** [review]

**Pull request merged**